website/source/docs/providers/google/r/compute_backend_service.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "google"
+page_title: "Google: google_compute_backend_service"
+sidebar_current: "docs-google-resource-backend-service"
+description: |-
+  Creates a Backend Service resource for Google Compute Engine.
+---
+
+# google\_compute\_backend\_service
+
+
+
+
+## Example Usage
+
+```
+resource "google_compute_backend_service" "foobar" {
+    name = "blablah"
+    health_checks = ["${google_compute_http_health_check.default.self_link}"]
```

Comment:
I guess the plan is to create instance group managers then pull out their computed instance_group attributes to build a list here?  It probably makes sense to include that in this example.